core/src/main/java/com/zextras/carbonio/files/message_broker/interfaces/MessageBrokerManager.java

```diff
@@ -0,0 +1,15 @@
+// SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package com.zextras.carbonio.files.message_broker.interfaces;
+
+import com.zextras.carbonio.message_broker.MessageBrokerClient;
+import com.zextras.carbonio.message_broker.events.generic.BaseEvent;
+
+public interface MessageBrokerManager {
+  boolean healthCheck();
+  void startAllConsumers();
+  void publishEvent(BaseEvent event);
+  MessageBrokerClient getMessageBrokerClient();
```

Comment:
Add javadoc to these methods. It will be inherited by every concrete classes that will implement this interface.
Move here some generic javadoc added in the concrete class